### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by TebbeUbben**
Extracted missing strings from preferences files and translated them to German.

### CODE REVIEW

File: app/src/main/res/values-de/strings-de.xml [hunk 1]
```diff
@@ -485,8 +485,345 @@
...
+	<string name="vehicle_motion_extras">Aktiviere Extrafunktionen, wenn der Fahrzeugmodus erkannt wurde</string>
+	<string name="vibrate_alerts">Bei Alarmen vibrieren</string>
+	<string name="vibrate_scanning_status">Vibriere um den Scanstatus anzuzeigen</string>
+	<string name="volume_buttons_snooze">Das Drücken der Lautstärkeknöpfe snoozed einen aktiven Alarm in der App</string>
+	<string name="watch_vibrate_active_alerts">Lass die Uhr bei aktiven Glukosealarmen vibrieren</string>
+	<string name="widget_range_lines">Widget-Zielbereichslinien</string>
+	<string name="wifi_name_must_match">WLAN-Name muss übereinstimmen</string>
+	<string name="xdrip_motion_tracking">xDrip+ Bewegungserkennung</string>
+
+    <string-array name="SendToPebbleChoice">
```
Comment (jamorham) [inline app/src/main/res/values-de/strings-de.xml]: I think the array entries should be moved in to `values-de/arrays-de.xml`

**Comment by jamorham** [comment]
Hi there,

Thanks for submitting this. 

Strings and Language entries require careful checking because their impact isn't always as clear as can be with code.

I am in the process of reviewing this, but can you confirm that where the english hardcoded strings have been moved to `strings.xml` that these have been exactly copied without any modifications? 

If that is the case then it reduces the amount of checking I need to do to only the german texts.

**Comment by TebbeUbben** [comment]
I used the Alt-Enter method for string extraction, so there should be no modifications.

I also figured out that there are still some missspellings which I am going to remove today.

File: app/src/main/res/values-de/strings-de.xml [hunk 2]
```diff
@@ -485,8 +485,306 @@
...
+	<string name="todays_average_value">Heutiger Durchschnittswert</string>
+	<string name="total_carbs">Gesamtkohlenhydrate</string>
+	<string name="total_insulin">Gesamtinsulin</string>
+	<string name="translation_credits">Vielen Dank an unsere Übersetzer:\n\nPietballing (ro) :: 458\nAlino1963 (it) :: 432\nMilos Kozak (cs) :: 419\nTixastronauta (pt) :: 411\nSomyeong\'s mom (ko) :: 396\nJonny Johnson (de) :: 321\nCyrille David (fr) :: 286\nMaciej Glabik (pl) :: 268\nEcorman (es) :: 156\nVladislav Lebedev (ru) :: 130\nAdrianLxM (de) :: 063\nAlexandre Silva (pt) :: 006\nTimMoritz (de) :: 001</string>
+	<string name="transmitter_id">ID deines Dexcom Transmitters, z.B. 12AB3</string>
+	<string name="transmitter_pl_support">Transmitter (PL) Unterstützung</string>
+	<string name="turn_bluetooth_on">Schalte Bluetooth ein</string>
+	<string name="unbond_g5_before_read">Das G5 vor jedem Lesen unbounden.</string>
+	<string name="upload_to_dexcom">Aktiviere dies zum Hochladen auf Dexcoms Server.</string>
+	<string name="upload_to_dexcom_share">Lade Blutzuckerwerte als Dexcom Share hoch.</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings-de.xml]: als -> zu/nach...

File: app/src/main/res/values-de/strings-de.xml [hunk 3]
```diff
@@ -485,8 +485,306 @@
...
+	<string name="show_short_calibration_data">Zeige die Steigung und den Anfangswert in kurzer Form.</string>
+	<string name="show_standard_deviation">Zeige Standardabweichung der Werte.</string>
+	<string name="show_test_features">Zeige Extra- und Alpha-Testfunktionen</string>
+	<string name="show_total_carbs">Zeige die Gesamtmenge an Behandlungskohlenhydraten.</string>
+	<string name="show_total_insulin">Zeige die Gesamtmenge an Behandlungsinsulin.</string>
+	<string name="skip_lan_uploads">LAN-Uploads überspringen</string>
+	<string name="skip_local_lan">Überspringe Uploads für lokale Server mit 192.168.x.x Adressen, wenn keine Verbindung zu einem lokalen Netzwerk besteht.</string>
+	<string name="snoozes_will_silence_all">Snoozes werden als Mobilteile in der Gruppe, die die Akzeptierungs-Einstellung unten gesetzt haben, verstummen lassen.</string>
+	<string name="speak_glucose_twice">Sprich den Glukosewert jedesmal zwei mal</string>
+	<string name="speak_readings_shortcut">Werte-Sprechen-Verknüpfung</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings-de.xml]: Hier würde ich "Vorlesen" verwenden. Wie nennt Google das generell?
"Text in Sprache" gefällt mir aber auch nicht ;)

File: app/src/main/res/values-de/strings-de.xml [hunk 4]
```diff
@@ -485,8 +485,306 @@
...
+	<string name="repeat_notification_every_90_minutes">Wiederhole den Benachrichtigungston alle 90 Minuten, falls immer noch im Fahrzeugmodus.</string>
+	<string name="repeat_sound">Ton wiederholen</string>
+	<string name="repeatedly_restart_collection_service">Wiederholt den Kollektor-Dienst neustarten bei Anzeichen von verpassten Werten. Aktiviere dies nur, wenn du Datenverluste erfährst.</string>
+	<string name="reraise_before_snooze">Erhebe den Alarm erneut vor der Snooze-Zeit.</string>
+	<string name="reraise_not_snoozed_sooner">Erhebe den Alarm erneut, falls nicht früher gesnoozed.</string>
+	<string name="reset_bluetooth">Setze Bluetooth zurück, in dem es an und wieder ausgeschaltet wird, wenn keine Daten innerhalb von 20 Minuten empfangen werden.</string>
+	<string name="reset_bluetooth_g5">Setze Bluetooth zurück, in dem es an und wieder ausgeschaltet wird, damit die G5-Datenquelle funktioniert. Ohne dies wird der G5-Kollektor vielleicht versagen.</string>
+	<string name="reset_bluetooth_on_off">Setze Bluetooth zurück in dem es alle paar Minuten and und wieder ausgeschaltet wird! Wähle dies nur zum Testen aus. Dies kann Anderes von der Bluetooth-Nutzung abhalten.</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings-de.xml]: "zurück in dem " -> "zurück, indem "

File: app/src/main/res/values-de/strings-de.xml [hunk 5]
```diff
@@ -485,8 +485,306 @@
...
+	<string name="pref_message_influxdb_username">Benutzername für die Authentifikation, Es ist nicht empfohlen das Nutzen ohne Authentifikation zu erlauben.</string>
+	<string name="pref_node_wearG5">Das Gerät auf dem der Kollektordienst läuft.</string>
+	<string name="pref_ringtone_silent">Lautlos</string>
+	<string name="pref_summary_disable_wearG5_on_lowbattery">Deaktiviere das Erzwingen des Wear-Collection-Dienstes beim Batterie-Niedrig-Alarm der Uhr.</string>
+	<string name="pref_summary_disable_wearG5_on_missedreadings">Deaktiviere das Erzwingen des Wear-Collection-Dienstes bei Minuten verpasster Werte.</string>
+	<string name="pref_summary_enable_wearG5">Verbinde die Uhr zum BT-Kollektor, wenn das Telefon außer Reichweite ist.</string>
+	<string name="pref_summary_force_wearG5">Erzwinge das Telefon, den Wear-Collection-Dienst zu benutzen.</string>
+	<string name="pref_summary_influxdb_enabled">Falls du eine private InfluxDB Datenbank benutzt, sollte dies aktiviert sein.</string>
+	<string name="pref_summary_node_wearG5">Erzwinge das Telefon, diesen Wear-Collection-Dienst zu benutzen.</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings-de.xml]: "Erzwingen" -> "Zwingen" ... kommt mehrfach vor.

File: app/src/main/res/values-de/strings-de.xml [hunk 6]
```diff
@@ -485,8 +485,306 @@
...
...s zu tun ist, Sie aber dennoch den Import benutzen wollen, fragen Sie einen erfahrenen Benutzer.\n\n\n        Anleitung:\n\n        Platzieren Sie die Datenbank, die Sie installieren wollen, im Ordner \\\'xdrip\\\' auf dem primären Speicher\n        (manchmal auch \\\'interner Speicher\\\' vom Dateiexplorer genannt)\n        Das ist derselbe Ordner in den xDrip auch exportiert.\n        \n        Falls Sie eine eine ZIP-Datei haben, entpacken Sie sie (normalerweise lang drücken->entpacken)\n        Falls sie in einem Unterordner entpackt werden sollte, wird sie immernoch gefunden werden.\n \n        Nachdem die Datei platziert ist, starten Sie die Import-Routine.\n        Scrollen Sie zum korrekten Datenbanknamen, klicken Sie darauf und folgen Sie den Anweisungen auf dem Bildschirm.\n \n \n        WARNUNG: Eine Datenbank kann nur importiert werden, wenn sie dieselbe Version hat, die momentan von xDrip benötigt wird.\n        Die Datenbankversionen werden sich mit zukünftigen xDrip Aktualisierungen vielleicht ändern.\n        Es ist am besten, wenn Sie die Datenbank mit derselben Version von xDrip importieren, mit der sie auch exportiert wurde, und xDrip dann aktualisieren.</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings-de.xml]: I can't find this string (`import_db_warning`) in the general strings.xml of this PR.

File: app/src/main/res/values-de/strings-de.xml [hunk 7]
```diff
@@ -485,8 +485,306 @@
...
+	<string name="experimental_rdfuino_support">Experimentelle Unterstützung für den RFduino von Tomasz Stachowicz</string>
+	<string name="experimental_transmitter_fpv_uav">Experimentelle Unterstützung für den das \'Transmitter\' Gerät von @FPV-UAV - nur Test</string>
+	<string name="extra_calibration_options">Extra Kalibrierungsoptionen</string>
+	<string name="extra_logging">Extra Loggingeinstellugen</string>
+	<string name="falling_threshold">Fall-Warngrenze</string>
+	<string name="flair_colors">Flair-Farben</string>
+	<string name="follower_chime_new">Neuer-Follower-Läuten</string>
+	<string name="for_interacting_compatible_apps">Zum Interagieren mit anderen kompatiblen Apps</string>
+	<string name="force_g5_ui_thread">G5 auf den UI-Thread zwingen</string>
+	<string name="forcing_alternate_language">Erzwige momentan das Benutzen einer alternativen Sprache</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings-de.xml]: Ich glaube, das müsste "Erzwingt" heißen.

File: app/src/main/res/values-de/strings-de.xml [hunk 8]
```diff
@@ -485,8 +485,306 @@
...
+	<string name="call">Anrufen</string>
+	<string name="capture_percentage">Paket-Erfassungsrate</string>
+	<string name="carb_ratio">Kohlenhydratfaktor</string>
+	<string name="change_sensor_total_days">Sensor-Gesamttage ändern</string>
+	<string name="choose_use_treament_bg">Wähle aus, ob der Blutzucker im Behandlungseintrag zum Kalibrieren benutzt werden soll.</string>
+	<string name="chosse_language">Wähle eine bestimmte Sprache aus.</string>
+	<string name="close_gatt">Wenn der Bluetooth-Watchdog oft aktiviert wird, dann kannst du versuchen diese Option abzuwählen, um zu testen, ob es hilft.</string>
+	<string name="cloud_upload">Cloud-Upload</string>
+	<string name="collector_battery_alerts">Kollektor-Batteriealarme</string>
+	<string name="confirm_remote_snoozes">Einen Dialog zum Bestätigen von jedem Remote-Snozze erscheinen lassen.</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings-de.xml]: Snozze -> Snooze

File: app/src/main/res/values-de/strings-de.xml [hunk 9]
```diff
@@ -480,13 +480,310 @@
...
+	<string name="alert_calibration_request">Alarmieren, wenn eine Kalibrierung angefordert wird</string>
+	<string name="alert_minutes_reraise">Anzahl der Minuten, bevor derselbe Alarm nach dem Snoozen ertönt</string>
+	<string name="alert_noisy_values">Bei verrauschten Werten nach x Minuten alarmieren</string>
+	<string name="alert_repeat_minutes">Alarmwiederholungszeit</string>
+	<string name="alert_reraise_time">Alarmwiedererhebungszeit</string>
+	<string name="alert_seconds_reraise">Anzahl der Sekunden, die vergehen müssen, bevor derselbe Alarm erneut ertönt.</string>
+	<string name="alert_snooze">Alarm-Snooze</string>
+	<string name="alert_snoozed_by_watch">Alarm von der Uhr gesnoozed</string>
+	<string name="alert_sound">Alarmton</string>
+	<string name="all_new_glucose_data_plugin">\"ALLE neuen Glukosedaten werden vom ausgewählten Plugin und nicht vom xDrip-Algorithmus berechnet und gespeichert. Dies ist nicht getestet und kann ungenau und fehlerhaft sein. Nur unter extremer Vorsicht benutzen.</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings-de.xml]: Hier ist noch ein Schrägstrich und ein doppeltes Hochkomma zu viel?

File: app/src/main/res/values-de/strings-de.xml [hunk 10]
```diff
@@ -480,13 +480,310 @@
...
+	<string name="act_as_motion_master">Als Motion-Master arbeiten</string>
+	<string name="advanced_bluetooth_settings">Erweiterte Bluetootheinstellungen</string>
+	<string name="advanced_calibration">Erweiterte Kalibrierung</string>
+	<string name="advanced_g5_settings">Erweiterte G5-Einstellungen für seltene Situationen</string>
+	<string name="aggressive_service_restarts">Aggressive Dienstneustarts</string>
+	<string name="alert_calibration_request">Alarmieren, wenn eine Kalibrierung angefordert wird</string>
+	<string name="alert_minutes_reraise">Anzahl der Minuten, bevor derselbe Alarm nach dem Snoozen ertönt</string>
+	<string name="alert_noisy_values">Bei verrauschten Werten nach x Minuten alarmieren</string>
+	<string name="alert_repeat_minutes">Alarmwiederholungszeit</string>
+	<string name="alert_reraise_time">Alarmwiedererhebungszeit</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings-de.xml]: Alarm-Abspiel-Intervall?
It is hard to find anything that is understandable, not the same as "repeat" and not a longer explaination what the re-raise interval is. Hm?

File: app/src/main/res/values-de/strings-de.xml [hunk 11]
```diff
@@ -480,13 +480,310 @@
...
 	<string name="short_on_text_for_switches">AN</string>
 	<string name="install_pebble_apps">Installieren Sie Pebble Apps</string>
-	<string name="sliding_24_hour_window_summary">Verwenden Sie den letzten 24 Stunden anstatt Zeit seit Mitternacht für die Statistik.</string>
+	<string name="sliding_24_hour_window_summary">Verwende die letzten 24 Stunden anstatt der Zeit seit Mitternacht für die Statistik.</string>
 	<string name="sliding_24_hour_window">Schiebefenster</string>
-	<string name="split">Teilt</string>
+	<string name="split">Teilen</string>
 	<string name="delete">Löschen</string>
+
+	<string name="ac1_estimation_dcct">A1c-Vorhersage im DCCT-Format (%)</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings-de.xml]: Im Deutschen wird in der Regel "HbA1c" verwendet.

File: app/src/main/res/values-de/strings-de.xml [hunk 12]
```diff
@@ -480,13 +480,310 @@
...
+	<string name="ac1_estimation_ifcc">A1c-Vorhersage im IFCC-Format (mmol/mol)</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings-de.xml]: Hier ebenso "HbA1c".

File: app/src/main/res/values-de/strings-de.xml [hunk 13]
```diff
@@ -480,13 +480,310 @@
 	<string name="predictive_model_inner_workings">Vorhersagemodell (Erweiterte Anzeige)</string>
 	<string name="noise_model_inner_workings">Rauschmodell (Erweiterte Anzeige)</string>
 	<string name="glucose_number_from_filtered">Glukose aus gefilterten Werten</string>
-	<string name="if_the_phone_has_text_to_speech">Sofern das Telefon über Text-in-Sprache verfügt, wird jeder neue Wert vorgelesen.</string>
+	<string name="if_the_phone_has_text_to_speech">Sofern das Telefon über Text-zu-Sprache verfügt, wird jeder neue Wert vorgelesen.</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings-de.xml]: Google calls this "Text-in-Sprache". The other sounds more idiomatic to me as well, but I'd stick with the name the user can find in the phone's settings.

File: app/src/main/res/values-de/strings-de.xml [hunk 14]
```diff
@@ -204,10 +204,10 @@
...
 	<string name="send_data_to_android_wear_watch">Daten an Watchface senden</string>
 	<string name="android_wear_integration">Android Wear Integration</string>
-	<string name="pref_wear_connectG5">Schließen Sie Wear bis G5</string>
-	<string name="pref_summary_wear_connectG5">Schließen Sie tragen direkt an G5</string>
-	<string name="pref_use_wear_connectG5">Verwendung Tragen G5 Collection Service</string>
-	<string name="pref_summary_use_wear_connectG5">Force-Telefon Wear G5 Collection Service zu verwenden,</string>
+	<string name="pref_wear_connectG5">Verbinde die Uhr mit dem G5</string>
+	<string name="pref_summary_wear_connectG5">Direkte Verbindung zum G5</string>
+	<string name="pref_use_wear_connectG5">Verwende den G5 Kollektor-Dienst</string>
+	<string name="pref_summary_use_wear_connectG5">Zwinge das Telefon, den Wear-Collection-Dienst zu benutzen</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings-de.xml]: Ha, you found some failed google translations :+1: 
But I can't find them in the general strings.xml. I guess (hope!) those have never been used :)

**Comment by jamorham** [comment]
This looks good, I will merge it for wider testing.

Thanks @TebbeUbben for providing the translations and @AdrianLxM for the review.

**Comment by TebbeUbben** [comment]
Seems like there are still english texts appearing but I have extracted and translated the corresponding string :/

**Comment by jamorham** [comment]
Can you provide an example?

**Comment by TebbeUbben** [comment]
https://drive.google.com/file/d/0B21CovNrIJ4PbDJhSld5cWhQYXM/view?usp=drivesdk

**Comment by jamorham** [comment]
Which version are you using? Try the April 5th Nightly.

**Comment by TebbeUbben** [comment]
Ok, seems to be working now with the 5th Nightly. Thanks!

**Pull request merged**